.vib/aspnet-core/goss/goss.yaml

```diff
@@ -0,0 +1,15 @@
+file:
+  /app:
+    exists: true
+    filetype: directory
+    mode: "0777"
+    owner: root
+    group: root
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: true
+    filetype: directory
+    mode: "1777"
+command:
+  dotnet-cli:
```

Comment:
I don't think this test adds any value to the chart verification. I would say checking this binary exists something to be tested in the container's test suite, wdyt?

Reply:
Agreed, thx. Updating it now.